com.github.k4zmu2a.spacecadetpinball.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be submitted and maintained upstream.

Reply:
https://github.com/k4zmu2a/SpaceCadetPinball/pull/111

Reply:
Merged